app/tests/v1/test_v1.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import unittest
+import json
+
+#local
+LOCALPATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, LOCALPATH + '/../../../')
+
```

Comment:
@danuluma What's the purpose of line 7 & 8?

Reply:
it sets the root of the project to where the app folder(?module) is....It basically takes the current location and moves back 3 times the sets that as the root(project root/base)....thus for any module I import here, I only have to specify its location from the project base,,,,, I had problems when importing modules whereby modules were not being found

Reply:
and after researching a bit, that was the solution I got,,,and it worked